src/api/app/controllers/webui/user_controller.rb

```diff
@@ -73,14 +74,15 @@ def save
     @displayed_user = User.find_by_login(params[:user][:login])
 
     unless User.current.is_admin?
-      if User.current != @displayed_user
+      if User.current != @displayed_user || !@configuration.accounts_editable?
```

Comment:
@ChrisBr as discussed, we don't need to add the second check inside the `unless` (which I dropped just right now). It's already covered by ` || !@configuration.accounts_editable?`